src/model/mmkv.ts

```diff
@@ -6,6 +6,7 @@ export const STORAGE_IDS = {
   DOMINANT_COLOR: 'DOMINANT_COLOR',
   EXPERIMENTAL_CONFIG: 'EXPERIMENTAL_CONFIG',
   IMGIX_CACHE: 'IMGIX_CACHE',
+  LOCAL_STORAGE: 'LOCAL_STORAGE',
```

Comment:
Not sure what would be a better name since it's literally our "localStorage" (the global available storage object). at least all the references in code use localStorage name. but yeah I'm open for discussion

Reply:
Sorry I should have been more clear 😛 I just mean like, it's specific to Android atm. Would it be OK if we reused this with another MMKV instance elsewhere in the app? If not, I was thinking like `ANDROID_LOCAL_STORAGE`. Small nit though, not a huge deal ✌️

Reply:
I think id rather call it `LOCAL_STORAGE_ADAPTER`?

Reply:
It will be eventually used on both iOS and Android, I'm gonna use this for android only for now, so no time has been spent on fixing something that just works on iOS anyway.

LOCAL_STORAGE_ADAPTER might be a better name, thanks